physical/physical_test.go

```diff
@@ -283,7 +283,7 @@ func testBackend_ListPrefix(t *testing.T, b Backend) {
 
 func testHABackend(t *testing.T, b HABackend, b2 HABackend) {
 	// Get the lock
-	lock, err := b.LockWith("foo", "bar")
+	lock, err := b.LockWith("/core/lock/foo", "bar")
```

Comment:
Actually, it doesn't do locking under that namespace; core/lock is the actual lock path :-)